### DISCUSSION TITLE
Add a reorder feature

### DISCUSSION THREAD

**Discussion by MiroslavShard** [community/community#13278] [Other Feature Feedback, Questions, & Ideas]
Very cool update! I love this new lists! It would be cooler if we could reorder them by dragging, like pinned repos.

**Comment by andrei0x309** [comment]
Yeah, reorder is needed especially when people change focus from one language to another, or one technology to another, this space is very dynamic reordering is an absolute necessity for lists.

**Comment by nop** [comment]
Custom ordering will also help me find the list I'm looking for easier, as I can control where it appears.

**Comment by Lumitex** [comment]
> Very cool update! I love this new lists! It would be cooler if we could reorder them by dragging, like pinned repos.

**Comment by VladimirCreator** [comment]
It would be great to have `<select>` that lets sort the list in manual order, added date order, etc..

**Comment by Anhmarukita123** [comment]
Mình muốn học python bạn giúp mjnh

**Comment by Facebook979** [comment]
> Very cool update! I love this new lists! It would be cooler if we could reorder them by dragging, like pinned repos.

दिस साइट इज वेरी इजी फॉर करेंसी चेंज हिंदी कन्वर्ट माय अकाउंट इंडियन रुपीस

**Comment by yasinnaal** [comment]
I agree it will be good feature. you can have look on  **lists enhancement** feedback and share your thoughts.

https://github.com/orgs/community/discussions/59955
